prow/config.yaml

```diff
@@ -219,6 +219,11 @@ branch-protection:
         contexts:
         - cla/google
         - tide
+        - "Build (1.17.x, ubuntu-latest)"
```

Comment:
Oh I just realized this will change will apply to all repos in knative org, which is not correct since some of the repos do not enable these GitHub Actions.